app/views/layouts/_matomo_tag.html.haml

```diff
@@ -1,15 +1,24 @@
+
+- if Spree::Config.matomo_tag_manager_url.present?
+  :javascript
+    var _mtm = _mtm || [];
+    _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    var u="#{Spree::Config.matomo_tag_manager_url}";
```

Comment:
Would it be possible to use human-readable variable naming here instead of single-letter variables?

Reply:
We could do but I kind of like keeping it close to the provided embed code, as that way we can more easily see if it needs updating - eg that made it easier for me to spot that filenames of piwik had changed to matomo.